test/entity-store.js

```diff
@@ -143,6 +143,42 @@ suite('entity-store', function() {
 			));
 		});
 
+		suite('nofollow siren links differ from href with token', () => {
```

Comment:
Since we ignore the passed in token for nofollow links, they get cached under `''` instead of the passed in token's would-be cacheKey. As such if you try to `get` the same thing as an href directly it won't be in the store.